beacon-chain/das/availability_columns.go

```diff
@@ -112,23 +121,33 @@ func (s *LazilyPersistentStoreColumn) IsDataAvailable(
 	// ignore their response and decrease their peer score.
 	roDataColumns, err := entry.filter(blockRoot, blockCommitments)
 	if err != nil {
-		return errors.Wrap(err, "incomplete DataColumnSidecar batch")
+		return errors.Wrap(err, "filter")
+	}
+
+	// https://github.com/ethereum/consensus-specs/blob/dev/specs/fulu/p2p-interface.md#datacolumnsidecarsbyrange-v1
+	verifier := s.newDataColumnsVerifier(roDataColumns, verification.ByRangeRequestDataColumnSidecarRequirements)
```

Comment:
why only the `byRangeRequest` requirements ?

Reply:
Because `LazilyPersistentStoreColumn` is only used for columns retrieved with RPC by range requests.